### DISCUSSION TITLE
User Agent String

### DISCUSSION THREAD

**Discussion by questionlp** [darrenburns/posting#5] [Ideas]
By default, it looks like Posting does not send a User-Agent header value. It would be nice to have an option to send a string like `Posting/<version>` instead of adding it manually.

Some web application firewalls, including CloudFlare, will block requests if no User-Agent is set, as it could mark the request as coming from a bad or malicious bot.

**Comment by darrenburns** [comment]
Yup, good point. I was going to add this after having a means of showing headers that are attached automatically, but I'll just do it now to make things more useful.

**Comment by questionlp** [comment]
Great to hear and looking forward it seeing it. Really love the design and simplicity of the program!

**Reply by darrenburns** [reply]
Thanks! A User-Agent string should be getting included in requests now :)

**Reply by questionlp** [reply]
Just updated the local copy of the repo and it's looking good. Thanks for the quick turnaround

![image](https://github.com/darrenburns/posting/assets/13372595/0840630a-6d87-48ea-81d0-9865bcece2ae)
